CONTRIBUTING.md

```diff
@@ -502,3 +494,12 @@ go: github.com/golangci/golangci-lint@v1.31.0 requires
 `go env GOPROXY` should return `https://proxy.golang.org,direct`. If it does not, set it as an environment variable:
 
 `export GOPROXY=https://proxy.golang.org,direct`
+
+## Exceptions and changes in the rules
+
+While the ruling on this and other documents in this repository is what we strive to follow, we acknowledge that rules may be 
+unfeasible to be applied in some situations or that procedures may need to change over time. Exceptions or changes to the rules 
```

Comment:
```suggestion
unfeasible to be applied in some situations. Exceptions to the rules 
```

We should separate changes from exceptions.  I'd like to avoid a situation where there is a disconnect between expectations where some may expect that they are discussing an exception and others think they are discussing a permanent change.

Reply:
Good feedback.

Reply:
That's good feedback, thanks, I also changed the title to be in line with this.